kvdb-rocksdb/src/lib.rs

```diff
@@ -905,7 +905,7 @@ rocksdb.db.get.micros P50 : 2.000000 P95 : 3.000000 P99 : 4.000000 P100 : 5.0000
 		assert!(settings.contains(" block_size: 323232"));
 
 		// LRU cache (default column)
-		assert!(settings.contains("block_cache_options:\n    capacity : 8388608"));
+		assert!(settings.contains("block_cache_options:\n    capacity : 115343360"));
```

Comment:
I haven't checked why it changed here, but this should be OK.

Reply:
The cache size was probably changed? Can you maybe check that in the changelogs?

Reply:
The change may relate to https://github.com/facebook/rocksdb/blob/main/HISTORY.md#public-api-changes-10

I can dig more if you think this is important